.changie.yaml

```diff
@@ -5,15 +5,15 @@ versionHeaderPath: ""
 changelogPath: CHANGELOG.md
 versionExt: md
 versionFormat: '## dbt-core {{.Version}} - {{.Time.Format "January 02, 2006"}}'
-kindFormat: '### {{.Kind}}'
-changeFormat: '- {{.Body}} ([#{{.Custom.Issue}}](https://github.com/dbt-labs/dbt-core/issues/{{.Custom.Issue}}), [#{{.Custom.PR}}](https://github.com/dbt-labs/dbt-core/pull/{{.Custom.PR}}))'
+kindFormat: "### {{.Kind}}"
+changeFormat: "- {{.Body}} ([#{{.Custom.Issue}}](https://github.com/dbt-labs/dbt-core/issues/{{.Custom.Issue}}), [#{{.Custom.PR}}](https://github.com/dbt-labs/dbt-core/pull/{{.Custom.PR}}))"
```

Comment:
Can we merge the updates to `.changie.yml` in a separate PR?

Reply:
looks like some extra formatting changes crept in here, removed those and just kept adding my user name